aptos-move/aptos-release-builder/src/components/gas.rs

```diff
@@ -46,10 +46,17 @@ pub fn generate_gas_upgrade_proposal(
             generate_blob(writer, &gas_schedule_blob);
             emitln!(writer, ";\n");
 
-            emitln!(
-                writer,
-                "gas_schedule::set_gas_schedule(framework_signer, gas_schedule_blob);"
-            );
+            if next_execution_hash.is_empty() {
+                emitln!(
```

Comment:
What’s the diff between the if else cases here?

Reply:
else has `&` before the `framework_signer`.

the single-step generation had something like `let framework_signer = &core_signer;` so that their `framework_signer` is of type `&signer`, but for multi-step the `framework_signer` is of type `signer`. I can update this part to be more consistent across the board